terraform/deployments/cluster-services/aws_auth_configmap.tf

```diff
@@ -27,6 +27,20 @@ locals {
       groups   = ["cluster-admins"]
     }
   ]
+
+  # only the test environment has a CI planner user to check commits before merge into main
```

Comment:
Why is this specific to the test account? Surely we'd want CI to be able to show the plan for all the environments?

Reply:
In fact even if it didn't need to talk to the other account, there's still no need to make this environment-specific. The actual access is controlled via IAM - this is just the mapping, which is fine to exist in all the clusters.

Reply:
currently, the concourse planner is in test environment only: https://github.com/alphagov/govuk-infrastructure/blob/df299404ebb36f99c1e1c7671b5696ae58e88dc6/terraform/deployments/concourse-iam/main.tf#L70

The concourse CI planner that checks commits runs in the test environment only.

I didn't want to add users that will "never" be used in the other environments.

Reply:
Please keep the environments the same. This module shouldn't need to know the environment.

Reply:
Ok, I can do that.